applications/AISub.md

```diff
@@ -0,0 +1,256 @@
+  - The platform offers real-time notifications, so users can be instantly notified when the model execution is complete and can view detailed execution results and related statistics.
+
+
+
+#### Supplier Product Overview
+
+![AISub_Timeline](https://raw.githubusercontent.com/AISubtrate/AIsubBuild/master/AISub_Supplierwork.jpg)
+
+
+On the AISub platform, suppliers (model providers) can easily manage and publish their AI models through a specially designed interface and functional modules. Here are the key features available for suppliers:
+
+1. **Create Model**:
+  - Suppliers can create and publish new AI models through the platform. During model creation, suppliers need to provide detailed information about the model, such as model name, description, input/output format, intended use, etc.
+  - The platform supports uploading model files or referencing external models via links, ensuring that suppliers can flexibly publish their model resources.
+  - After creating a model, suppliers can choose whether to make it public (available to all users) or private (available to specific users only), catering to different application scenarios and privacy needs.
+  - **When using open-source models**, suppliers must include the corresponding open-source license. This ensures compliance and respects the contributions of the open-source community. Suppliers must clearly specify the type of license used and upload the relevant license files to the platform during the model's publication.
+
+2. **Create Version**:
+  - To support continuous updates and improvements, the platform allows suppliers to create multiple versions of an already published model. Each version can have different enhancements and updates, with detailed records of the differences between versions.
+  - When creating a new version, suppliers can describe the updates, improved features, and any changes that may impact model performance.
+  - Users can choose to use a specific version of a model when invoking it, ensuring that they always have access to the version that best suits their needs.
+
+3. **Pricing and Charging**:
+  - Suppliers can set usage fees for their models. The platform offers various pricing strategies, allowing suppliers to set fees based on usage frequency, time, computational resources consumed, and other factors.
+  - Suppliers can choose between fixed pricing or dynamic pricing models. Dynamic pricing allows for adjustments based on model demand and market conditions, enabling more flexible pricing strategies.
+  - **When using open-source models**, suppliers must include the corresponding open-source license. This ensures compliance and respects the contributions of the open-source community. Suppliers must clearly specify the type of license used and upload the relevant license files to the platform during the model's publication.
```

Comment:
We require the upload of license information because sometimes suppliers use open-source models, and it's important to ensure that these models comply with their respective open-source licenses, such as restrictions on commercial use. While we cannot fully verify the compliance of every model used by suppliers, including this information helps mitigate potential copyright and legal risks for the platform. We understand that this may rely on suppliers' honesty, but we aim to provide a safeguard. We are open to making this an optional field in the future.